src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -9628,3 +9629,140 @@ void ObjectMgr::LoadSceneTemplates()
 
     TC_LOG_INFO("server.loading", ">> Loaded %u scene templates in %u ms.", count, GetMSTimeDiffToNow(oldMSTime));
 }
+
+void ObjectMgr::LoadAreaTriggerTemplates()
+{
+    uint32 oldMSTime = getMSTime();
+    _areaTriggerTemplateStore.clear();
```

Comment:
you can not clear this storage on reload since areatriggers have a pointer to container element